### DISCUSSION TITLE
Upgrading 1.33 to 2.0.9

### DISCUSSION THREAD

**Discussion by elray** [kimai/kimai#4727] [On-Premise: Installation & Updates] [answered]
Hi

During my upgrade from 1.33 to 2.0.9 I am running the command:

`composer install --optimize-autoloader -n`

However this errors with:

> In ArrayNode.php line 304: Unrecognized option "translation" under "kimai.theme.branding". Available options are "company", "logo", "mini", "title".

This is caused by the following in my local.yaml file.

> theme:
        branding:
            translation: mybrandtranslation

Has this option gone/been replaced in Kimai 2.0.9 ?

regards
Eliot

**Comment by kevinpapst** [comment]
It was deleted, see https://github.com/kimai/kimai/blob/main/UPGRADING.md

**Reply by elray** [reply, marked as answer]
OK, missed that thanks Kevin. Will delete local.yaml
